setup.py

```diff
@@ -29,6 +29,8 @@
         install_requires=[
             "incremental",
             "requests >= 2.1.0",
+            # Require a recent Hyperlink as some old versions have serious bugs.
```

Comment:
I don't think you need a comment here, especially one that includes a curséd word like "recent" which will instantly become super stale. We directly depend on hyperlink now (hooray!) so it's just a requirement.